rudder-webapp/SOURCES/Makefile

```diff
@@ -47,10 +47,9 @@ localdepends: ./rudder-sources ./maven2/bin/mvn ./rudder-users.xml ./rudder-doc
 
 ./rudder-doc: /usr/bin/wget
 	# Get user manual from rudder-project.org and rename directory containing it to rudder-doc
-	$(WGET) -P ./rudder-doc/html -r -np -nH --cut-dirs=1 http://www.rudder-project.org/rudder-doc-${RUDDER_MAJOR_VERSION}/rudder-doc.html
-	mkdir -p ./rudder-doc/pdf
-	# Move the generated PDF from html directory to pdf
-	mv ./rudder-doc/html/rudder-doc.pdf ./rudder-doc/pdf/
+	$(WGET) -P ./rudder-doc/html -r -np -nH --cut-dirs=1 http://www.rudder-project.org/doc-${RUDDER_MAJOR_VERSION}/index.html
```

Comment:
Please don't include the index.html at the end of the URL. I'd rather use a standard URL and let the webserver serve whatever it thinks (aka what we configure) is best.